runhouse/resources/hardware/sky_ssh_runner.py

```diff
@@ -303,10 +303,6 @@ def terminate(self):
                 )
             )
 
-            self.tunnel_proc = None
-            self.local_bind_port = None
-            self.remote_bind_port = None
-
             if "ControlMaster" not in port_fwd_cmd:
                 return
```

Comment:
should we also set them to none here too? added this case for docker cluster where the cancel_port_fwd command doesn't work and the tunnel seems to be terminated upon exiting already

Reply:
ope ya missed that